core/mondoo-linux-security.mql.yaml

```diff
@@ -238,6 +238,8 @@ queries:
   - uid: mondoo-linux-security-filesystem-integrity-is-regularly-checked
     title: Ensure filesystem integrity is regularly checked
     impact: 50
+    filters: |
+      asset.name != "almalinux:8.9" || asset.name != "almalinux:9.3" || asset.name != "centos:7" || asset.name != "centos:8" || asset.name != "registry.access.redhat.com/ubi8/ubi:8.0-122" || asset.name != "registry.access.redhat.com/ubi8/ubi:8.9-1107"
```

Comment:
Instead of using specific asset names we should use `asset.kind == "container-image"`. The current approach does not work for forks or custom build containers